### ISSUE TITLE
`rake setup_env` issue

### ISSUE THREAD

**Issue by gmassanek** [DataDog/integrations-extras#19]
I'm getting an error setting up the environment. Looks like an underlying gem is failing?

```
geoff:integrations-extras geoffrey$ rake setup_env
/Users/geoffrey/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/datadog-sdk-testing-0.4.2/lib/tasks/sdk.rake
rake aborted!
NoMethodError: undefined method `last_comment' for #<Rake::Application:0x007fa970958b18>
/Users/geoffrey/dev/integrations-extras/Rakefile:21:in `load'
/Users/geoffrey/dev/integrations-extras/Rakefile:21:in `<top (required)>'
/Users/geoffrey/.rbenv/versions/2.2.2/bin/bundle:23:in `load'
/Users/geoffrey/.rbenv/versions/2.2.2/bin/bundle:23:in `<main>'
(See full trace by running task with --trace)
```

**Comment by lastmikoi** [comment]
Ran into the same issue, apparently one of the underlying gem is making use of a gone feature of the `rake` dependency.

Following https://stackoverflow.com/questions/35893584/nomethoderror-undefined-method-last-comment-after-upgrading-to-rake-11#35893625 I edited the Gemfile to use a pre-11.0 `rake` and the `setup_env` step seems to be working again.

**Comment by jeremy-lq** [comment]
Closing, as this issue was fixed by PR#24.
